docs/src/main/paradox/compatibility-guidelines.md

```diff
@@ -113,25 +113,25 @@ Java
 ## Versioning and Compatibility
 
 Starting from version 10.1.0, there will be two active release branches:
-- The "current" release line (in `main`), where we will basically continue to evolve Akka HTTP in the same way as currently. New features will be introduced here incrementally.
+- The "current" release line (in `main`), where we will basically continue to evolve Apache Pekko HTTP in the same way as currently. New features will be introduced here incrementally.
 - The "previous" release line (in a release-10.x branch), where the focus is on stability. We will continue to maintain the previous release by fixing serious bugs but it will not see new features. Previous releases will see less frequent releases over time.
 
 It is planned to rotate versions in an annual fashion. Meaning a new minor version will be created every year.
 Whenever a new minor version is created, it will move the at that point current minor version release series over into maintenance mode, making it the "previous".
 The former "previous" release has reached its end of life at this point. This way every release line is supported for two years.
 
-The Akka HTTP Team currently does not intend to break binary compatibility, i.e. no binary incompatible 11.x.y release is currently planned.
+The Apache Pekko HTTP Team currently does not intend to break binary compatibility, i.e. no binary incompatible 11.x.y release is currently planned.
     
 ## Specific versions inter-op discussion
 
-In this section we discuss some of the specific cases of compatibility between versions of Akka HTTP and Akka itself.
+In this section we discuss some of the specific cases of compatibility between versions of Apache Pekko HTTP and Akka itself.
 
 For example, you may be interested in those examples if you encountered the following exception in your system when upgrading parts 
-of your libraries: `Detected java.lang.NoSuchMethodError error, which MAY be caused by incompatible Akka versions on the classpath. Please note that a given Akka version MUST be the same across all modules of Akka that you are using, e.g. if you use akka-actor [2.5.3 (resolved from current classpath)] all other core Akka modules MUST be of the same version. External projects like Alpakka, Persistence plugins or Akka HTTP etc. have their own version numbers - please make sure you're using a compatible set of libraries.`
+of your libraries: `Detected java.lang.NoSuchMethodError error, which MAY be caused by incompatible Akka versions on the classpath. Please note that a given Akka version MUST be the same across all modules of Akka that you are using, e.g. if you use akka-actor [2.5.3 (resolved from current classpath)] all other core Apache Pekko modules MUST be of the same version. External projects like Alpakka, Persistence plugins or Apache Pekko HTTP etc. have their own version numbers - please make sure you're using a compatible set of libraries.`
```

Comment:
Missed an `Akka` here

Reply:
thank - I think I've fixed up the issues you raised - I'll double cjeck later

there is also at least 1 test issue that I need to debug